lib/sql.go

```diff
@@ -0,0 +1,52 @@
+package lib
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/qri-io/qri/sql"
+)
+
+// SQLMethods encapsulates business logic for the qri search command
+// TODO (b5): switch to using an Instance instead of separate fields
+type SQLMethods struct {
+	inst *Instance
+}
+
+// NewSQLMethods creates SQLMethods from a qri Instance
+func NewSQLMethods(inst *Instance) *SQLMethods {
+	return &SQLMethods{inst: inst}
+}
+
+// CoreRequestsName implements the requests interface
+func (m SQLMethods) CoreRequestsName() string { return "sql" }
+
+// SQLExecParams defines paremeters for the exec Method
+// ExecParams provides parameters to the execute command
+type SQLExecParams struct {
+	Query        string
+	OutputFormat string
+}
+
+// Exec runs an SQL query
+func (m *SQLMethods) Exec(p *SQLExecParams, results *[]byte) error {
+	if m.inst.rpc != nil {
+		return m.inst.rpc.Call("SQLMethods.Exec", p, results)
```

Comment:
We should wrap this with better error handling for more user friendliness.

Also error handling in general - it's probably going to reach the user very often as we won't really support "all of SQL" out of the box and they will constantly be hitting edge cases.